.github/workflows/test_release_linux.yml

```diff
@@ -0,0 +1,46 @@
+name: Logging Test Linux pipeline
+
+on:
+  push:
+    branches:
```

Comment:
I guess you'd like a manual dispatch instead for this one

Reply:
see https://github.com/newrelic/infrastructure-agent/blob/master/.github/workflows/prerelease_linux.yml#L3-L8

Reply:
I think this is for test, so probably they want to have them always on push 🤔

Reply:
ah, got it, it is a test release.

Reply:
Yes, right now it's running always we push...   I will validate with the team if they prefer keep as is, run when we create a PR or even manual dispatch as you suggested